### PR TITLE
Support intl-tel-input 19

### PR DESCRIPTION
**Pull request by tlebreton**
Hi 

This PR upgrade intl-tel-input to version 19. (Don't think we need to touch at somethink else).

Thanks for the work on this libary :)

### CODE REVIEW

**Comment by mpalourdio** [comment]
Thanks for this PR. Looks like the new API is not backward compatible, tests are failing :/

**Comment by tlebreton** [comment]
Ok i will have a look

**Comment by tlebreton** [comment]
There is the last that fails but i don't know if it's normal or not.

File: src/lib/components/intl-tel-input.component.spec.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (mpalourdio) [inline src/lib/components/intl-tel-input.component.spec.ts:10]: good catch ! The linter should have warned...

File: src/lib/components/intl-tel-input.component.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export class IntlTelInputComponent implements AfterViewInit {
     @Output() private E164PhoneNumberChange = new EventEmitter<string | null>();
     @ViewChild('intlTelInput') private _inputElement!: ElementRef;
     private _phoneNumber!: string;
-    private _intlTelInput: any;
+    public _intlTelInput: any;
```
[the remark this answers is not in the document]
Comment (tlebreton) [reply src/lib/components/intl-tel-input.component.ts]: Oups my bad was debugging somethink and forget about it ....

**Comment by tlebreton** [comment]
I remove the test as you said :)

File: src/lib/components/intl-tel-input.component.spec.ts (hunk 1 above)
Comment (tlebreton) [reply src/lib/components/intl-tel-input.component.spec.ts:10]: Maybe updating all packages will help

**Comment by mpalourdio** [comment]
> I remove the test as you said :)

Thanks, But this should be removed too (and readme should be updated because this option is removed) :
- https://github.com/mpalourdio/intl-tel-input-ng/blob/6112f0727760d80840a27d034d1008559c017871/src/lib/components/intl-tel-input.component.ts#L28
- https://github.com/mpalourdio/intl-tel-input-ng/blob/6112f0727760d80840a27d034d1008559c017871/src/lib/components/intl-tel-input.component.ts#L36-L39
- https://github.com/mpalourdio/intl-tel-input-ng/blob/6112f0727760d80840a27d034d1008559c017871/src/lib/components/intl-tel-input.component.ts#L42-L44

**Comment by mpalourdio** [comment]
Good job. Will review and polish ASAP, before the end of the week in any case. Will need to squash your commits. I hadn't seen you had forked and committed to master, so will have to do this locally, because I think i won't be able to push/force push to your master :/ (not a big deal anyway)

Thanks a lot for this contribution!

**Comment by mpalourdio** [comment]
v0.11.0 has been released. Thanks for contributing!

**Pull request merged**